client/lib/abtest/active-tests.js

```diff
@@ -102,6 +102,16 @@ module.exports = {
 		defaultVariation: 'disabled',
 		allowExistingUsers: true,
 	},
+	coldStartReader: {
+		datestamp: '20160622',
+		variations: {
+			noEmailColdStart: 10,
+			noEmailNoColdStart: 10,
+			noChanges: 80
+		},
+		defaultVariation: 'noChanges',
+		allowExistingUsers: false,
+	}
```

Comment:
Missing comma after this I think.

Tests are failing on this line:

`client/lib/abtest/active-tests.js: Unexpected token (115:1)`